[0m08:30:13.268140 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106d8c090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10701c450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107034d50>]}


============================== 08:30:13.270834 | 694b3ff7-d40c-4780-a94a-e08037a1ee4c ==============================
[0m08:30:13.270834 [info ] [MainThread]: Running with dbt=1.6.1
[0m08:30:13.271130 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/Users/vinayreddysama/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt init snowflake_project', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m08:30:13.271457 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/Users/vinayreddysama/.dbt')}
[0m08:30:13.271615 [info ] [MainThread]: Creating dbt configuration folder at 
[0m08:30:13.272101 [debug] [MainThread]: Starter project path: /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/dbt/include/starter_project
[0m08:30:13.274975 [info ] [MainThread]: 
Your new dbt project "snowflake_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m08:30:13.275146 [info ] [MainThread]: Setting up your profile.
[0m08:30:13.275436 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m08:30:13.275735 [debug] [MainThread]: Command `dbt init` failed at 08:30:13.275678 after 0.02 seconds
[0m08:30:13.275902 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10701c750>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10701f010>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10701c450>]}
[0m08:30:13.276069 [debug] [MainThread]: Flushing usage events
[0m08:31:53.853718 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104cb0b50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1053ed990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1014e7190>]}


============================== 08:31:53.856243 | 34ed50d7-7a31-4de5-80ca-541a1317ee89 ==============================
[0m08:31:53.856243 [info ] [MainThread]: Running with dbt=1.6.1
[0m08:31:53.856560 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/vinayreddysama/.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init snowflake_project', 'send_anonymous_usage_stats': 'True'}
[0m08:31:53.857110 [info ] [MainThread]: A project called snowflake_project already exists here.
[0m08:31:53.857434 [debug] [MainThread]: Command `dbt init` succeeded at 08:31:53.857371 after 0.01 seconds
[0m08:31:53.857610 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1053edc10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105339fd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105441350>]}
[0m08:31:53.857793 [debug] [MainThread]: Flushing usage events
[0m09:15:35.131835 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1024a9350>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1052ee990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105300990>]}


============================== 09:15:35.134449 | 1ec3cb36-e8f2-4569-8499-6d5d5ed6a721 ==============================
[0m09:15:35.134449 [info ] [MainThread]: Running with dbt=1.6.1
[0m09:15:35.134754 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/vinayreddysama/.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run --profile snowflake_project', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m09:15:35.135103 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/vinayreddysama/project/fullscript_project_Anil/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m09:15:35.135418 [debug] [MainThread]: Command `dbt run` failed at 09:15:35.135359 after 0.01 seconds
[0m09:15:35.135591 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1053e9790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105047f10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103f214d0>]}
[0m09:15:35.135755 [debug] [MainThread]: Flushing usage events
